pkg/vc/issuer/credentials.go

```diff
@@ -71,7 +71,7 @@ func CreateManifestCredential(issuerName string, supportedContexts []string) ([]
 // ParseWalletResponse parses VP received from the wallet and returns the DIDConnect response.
 func ParseWalletResponse(vpBytes []byte) (*DIDConnectCredentialSubject, error) {
 	// TODO https://github.com/trustbloc/edge-adapter/issues/87 validate the signature
-	pres, err := verifiable.ParsePresentation(vpBytes, verifiable.WithDisabledPresentationProofCheck())
+	pres, err := verifiable.ParsePresentation(vpBytes, verifiable.WithPresDisabledProofCheck())
```

Comment:
uhm - I may have missed this

Reply:
should be done in a followup PR (ie. the issue referenced in the TODO) as this involves verifying the proof in the VP the wallet delivers to the issuer adapter.

Reply:
This PR is all about the RP Adapter signing the verifiable presentation enclosing the authZ VC that is sent to the Issuer Adapter as proof they have been authorized to access the resource.